data/migrations/V0087__operations_log_mv.sql

```diff
@@ -0,0 +1,51 @@
+-- Materialized View: public.ofec_operations_log_mv
+
+DROP MATERIALIZED VIEW IF EXISTS public.ofec_operations_log_mv;
+
+SET search_path = public;
+
+CREATE MATERIALIZED VIEW public.ofec_operations_log_mv AS 
+ SELECT o.sub_id,
+    o.status_num,
+    o.cand_cmte_id,
+    o.beg_image_num,
+    o.end_image_num,
+        CASE
+            WHEN f.form_tp IS NULL THEN regexp_replace(o.form_tp::text, 'A|N|T'::text, ''::text)::character varying
+            ELSE f.form_tp
+        END AS form_tp,
+    f.rpt_yr,
+    f.amndt_ind,
+    o.rpt_tp,
+    o.scan_dt,
+    o.pass_1_entry_dt,
+    o.pass_1_verified_dt,
+    o.pass_3_coding_dt,
+    o.pass_3_entry_done_dt,
+    o.receipt_dt,
+    o.beginning_coverage_dt,
+    o.ending_coverage_dt,
+    o.file_num,
+    o.prev_file_num
+   FROM staging.operations_log o
+     LEFT JOIN disclosure.f_rpt_or_form_sub f ON o.sub_id = f.sub_id
+  WHERE (o.form_tp::text <> ALL (ARRAY['RFAI'::character varying::text, 'ICI'::character varying::text, 'F1Z'::character varying::text, 'F2Z'::character varying::text])) AND o.beg_image_num IS NOT NULL AND (o.status_num = ANY (ARRAY[0::numeric, 1::numeric]))
+WITH DATA;
+
+ALTER TABLE public.ofec_operations_log_mv
+  OWNER TO fec;
+GRANT ALL ON TABLE public.ofec_operations_log_mv TO fec;
+GRANT SELECT ON TABLE public.ofec_operations_log_mv TO fec_read;
+
+CREATE INDEX ofec_operations_log_mv_status_num_idx1
```

Comment:
It looks like we need a `UNIQUE INDEX` in order to refresh materialized. When I ran `invoke create_sample_db` I got an error message: 

```
cannot refresh materialized view "public.ofec_operations_log_mv" concurrently
HINT:  Create a unique index with no WHERE clause on one or more columns of the materialized view.
 [SQL: 'REFRESH MATERIALIZED VIEW CONCURRENTLY ofec_operations_log_mv']
```

Reply:
not sure if we need a  unique index to refresh MV. I will look into this.

Reply:
We do. Happy to help if you get stuck!

Reply:
Since we switched to MV, i see that a unique index is required. I have one unique index in place now. Thanks for flagging this.